src/app/zap-templates/zcl/data-model/chip/energy-evse-cluster.xml

```diff
@@ -0,0 +1,235 @@
+    <field name="Wednesday" mask="0x08"/>
+    <field name="Thursday" mask="0x10"/>
+    <field name="Friday" mask="0x20"/>
+    <field name="Saturday" mask="0x40"/>
+    <field name="Away" mask="0x80"/>
+  </bitmap>
+
+  <enum name="StartOfWeekEnum" type="ENUM8">
+    <cluster code="0x0099"/>
+    <item name="Sunday" value="0x00"/>
+    <item name="Monday" value="0x01"/>
+    <item name="Tuesday" value="0x02"/>
+    <item name="Wednesday" value="0x03"/>
+    <item name="Thursday" value="0x04"/>
+    <item name="Friday" value="0x05"/>
+    <item name="Saturday" value="0x06"/>
+  </enum>
+
+  <enum name="EnergyTransferStoppedReasonEnum" type="ENUM8">
+    <cluster code="0x0099"/>
+    <item name="EvStopped"    value="0x00"  optional="false"/>
+    <item name="EvseStopped"  value="0x01"  optional="false"/>
+    <item name="Other"        value="0x03"  optional="false"/>
+  </enum>
+
+  <struct name="ChargingTargetStruct">
+    <cluster code="0x0099"/>
+    <item name="TargetTime"   type="INT16U"  optional="false"/>
+    <item name="TargetSoC"    type="INT8U"   optional="true"/>
+    <item name="AddedEnergy"  type="INT32U"  optional="true"/>
+  </struct>
+
+  <cluster>
+    <name>Energy EVSE</name>
+    <domain>Energy Management</domain>
+    <description>This cluster provides an interface for the management of Electric Vehicle Supply Equipment (EVSE).</description>
+    <code>0x0099</code>
+    <define>ENERGY_EVSE_CLUSTER</define>
+    <client init="false" tick="false">true</client>
+    <server init="false" tick="false">true</server>
+
+    <attribute side="server" code="0x0000" define="STATE"                       type="EvseStateEnum"              writable="false"                  optional="false" isNullable="true">State</attribute>
+    <attribute side="server" code="0x0001" define="SUPPLY_STATE"                type="SupplyStateEnum"            writable="false"                  optional="false" isNullable="false">SupplyState</attribute>
+    <attribute side="server" code="0x0002" define="FAULT"                       type="FaultStateEnum"             writable="false"                  optional="false" isNullable="false">Fault</attribute>
+    <attribute side="server" code="0x0003" define="ENABLE_CHARGE_TIME"          type="elapsed_s"                  writable="false"                  optional="false" isNullable="true">EnableChargeTime</attribute>
+    <attribute side="server" code="0x0004" define="ENABLE_DISCHARGE_TIME"       type="elapsed_s"                  writable="false"                  optional="false" isNullable="true">EnableDischargeTime</attribute>
+    <attribute side="server" code="0x0005" define="CIRCUIT_CAPACITY"            type="INT32U"  writable="false" default="0x0"    optional="false" isNullable="false">CircuitCapacity</attribute>
+    <attribute side="server" code="0x0006" define="MINIMUM_CHARGE_CURRENT"      type="INT32U"  writable="false" default="0x1770" optional="false" isNullable="false">MinimumChargeCurrent</attribute>
```

Comment:
This is confusing to me: spec seems to say default 6, but I have a clearly intentional value here. Something needs to change to make them match.

Reply:
It should be 6A, but the type changed to mA - so now needs to be 6000 as a default. 
I've also put in a spec PR to that affect.
https://github.com/CHIP-Specifications/connectedhomeip-spec/pull/7813

Reply:
Note that "default" in this XML and "default" in the spec are not the same thing in general @jamesharrow ...